torchaudio/csrc/decoder/README.md

```diff
@@ -0,0 +1,39 @@
+# Flashlight Decoder Binding
```

Comment:
This file can be put somewhere more explicit IMO. For example, CONTRIBUTING.md or in prototype directory.